src/test/kotlin/com/terraformation/backend/accelerator/model/SustainableDevelopmentGoalTest.kt

```diff
@@ -24,4 +24,60 @@ class SustainableDevelopmentGoalTest {
     assertEquals(
         SustainableDevelopmentGoal.Partnerships, SustainableDevelopmentGoal.forJsonValue("17"))
   }
+
+  @Test
+  fun `forJsonValue works correctly for names`() {
+    assertEquals(
+        SustainableDevelopmentGoal.NoPoverty, SustainableDevelopmentGoal.forJsonValue("NoPoverty"))
```

Comment:
Not convinced it's useful to cover every single enum value here rather than just testing whether name lookup works at all, but if we want full coverage, maybe something like this rather than listing the values all out explicitly?
```kotlin
SustainableDevelopmentGoal.entries.forEach {
  assertEquals(it, SustainableDevelopmentGoal.forJsonValue(it.name))
}
```